backend/utils/psyonix_api_handler.py

```diff
@@ -97,29 +99,30 @@ def get_rank_batch(ids, offline_redis=None):
     if 'detail' in data:
         return {str(i): {} for i in ids}
     for player in data:
-        rank_datas = []
+        rank_datas = {}
         unique_id = player['user_id']
-        names = {'13': 'standard', '11': 'doubles', '10': 'duel', '12': 'solo'}
+        names = {'13': 'standard', '11': 'doubles', '10': 'duel', '12': 'solo',
```

Comment:
We have enums that should already contain this mapping

Reply:
We don't have human-readable strings.

Reply:
Okay, but thats not required for this PR. We can do a PR where we change this globally.